nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -468,6 +468,20 @@ def _impl(inputs, attr, params):
             ignores=['index_type', 'T'])(new_inputs, attr)
     return _impl
 
+def _lrn():
```

Comment:
@siju-samuel 
I see few default params in tensorflow for LRN.

Please handle them here.